Fix C warning...

When build without netCDF logging, there is a warning:

```
cd /home/ed/pio4/ParallelIO/build/src/clib && /usr/local/bin/mpicc  -DCPRGNU -DHAVE_MPI -DINCLUDE_CMAKE_FCI -DLINUX -DTIMING -DUSE_PNETCDF_VARN -DUSE_PNETCDF_VARN_ON_READ -D_NETCDF -D_NETCDF4 -D_PNETCDF -I/home/ed/pio4/ParallelIO/src/clib -I/home/ed/pio4/ParallelIO/build/src/clib -I/usr/local/netcdf-4.4.1_mpich-3.2/include -I/usr/local/pnetcdf-1.8.1/include -I/home/ed/pio4/ParallelIO/src/gptl -I/home/ed/pio4/ParallelIO/build/src/gptl  -Wall -g    -std=c99 -o CMakeFiles/pioc.dir/pioc_support.c.o   -c /home/ed/pio4/ParallelIO/src/clib/pioc_support.c
/home/ed/pio4/ParallelIO/src/clib/pioc_support.c: In function ‘PIOc_set_log_level’:
/home/ed/pio4/ParallelIO/src/clib/pioc_support.c:104:9: warning: unused variable ‘ret’ [-Wunused-variable]
     int ret;
         ^~~
```

--- a/src/clib/pioc_support.c
+++ b/src/clib/pioc_support.c
@@ -101,13 +101,14 @@ int PIOc_strerror(int pioerr, char *errmsg)
  */
 int PIOc_set_log_level(int level)
 {
-    int ret;
 
 #if PIO_ENABLE_LOGGING
     /* Set the log level. */
     pio_log_level = level;
 
 #if NETCDF_C_LOGGING_ENABLED
+    int ret;
+    
     /* If netcdf logging is available turn it on starting at level = 4. */
     if (level > NC_LEVEL_DIFF)
         if ((ret = nc_set_log_level(level - NC_LEVEL_DIFF)))

